.github/compatibility-test-matrices/release-v8.5.x/ica-queries-chain-b.json

```diff
@@ -0,0 +1,21 @@
+{
+  "chain-a": [
+    "v7.7.0",
+    "v7.6.0",
+    "v7.5.0",
+    "release-v8.5.x"
+  ],
+  "chain-b": [
+    "v8.4.0",
```

Comment:
shouldn't this be in `chain-a` to be inverse of the `ica-queries-chain-a.json`?

Reply:
Yes, good catch! This was already wrong in release-v8.4.x, so I fixed it there too.